packages/offix-client/integration_test/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0",
   "description": "Integration tests for datasync",
   "scripts": {
-    "pretest": "kill $(lsof -t -i:4001) & node server/index.js &",
+    "pretest": "kill $(lsof -t -i:4001); kill $(lsof -t -i:4002); node server/index.js & node server/testx.js &",
```

Comment:
@darahayes I would like to initially keep both testx and the old server in order to smoothly migrate all tests, but this looks akwards for me, do you have any better idea?

Reply:
Sounds like a good start to me!